src/shared/canvas/DataExplorerGraphFlow/DateExplorerGraphFlow.tsx

```diff
@@ -1,10 +1,68 @@
-import React from 'react';
+import React, { useRef, useEffect, CSSProperties } from 'react';
```

Comment:
Something I was a bit confused about is that this PR includes changes for user stories 10 and 11. I thought each user story would be the most atomic unit of work that we will do, therefore 1 PR per user story. Maybe this is something we should discuss when we decide on how the user stories are going for us.

Reply:
It's okey to work on 2 user story in one PR, and it was already mentioned by Deneil, I can two PRs for this since the components are pretty much the same We can make it in one PR.